EIPS/eip-7883.md

```diff
@@ -0,0 +1,160 @@
+---
+eip: 7883
+title: ModExp Gas Cost Increase
+description: Increases cost of ModExp precompile
+author: Marcin Sobczak (@marcindsobczak), Marek Moraczyński (@MarekM25), Marcos Maceo (@stdevMac)
+discussions-to: https://ethereum-magicians.org/t/eip-7883-modexp-gas-cost-increase/22841
+status: Draft
+type: Standards Track
+category: Core
+created: 2025-02-11
+requires: 2565
+---
+
+## Abstract
+
+This EIP is modifying the `ModExp` precompile pricing algorithm introduced in [EIP-2565](./eip-2565.md).
+
+## Motivation
+
+There are cases where the `ModExp` precompile is underpriced for it's resource consumption. By modifying the `ModExp` pricing formula these scenarios would be covered with minimal impact on real world applications. The target is to make `ModExp` at least as fast as EcRecover precompile in all cases.
+
+## Specification
+
+As of `FORK_BLOCK_NUMBER`, the gas cost of calling the precompile at address `0x0000000000000000000000000000000000000005` will be calculated as follows:
```

Comment:
we don't refer hardfork number of timestamp now, you may say upon activation of this EIP, also forks are now timestamp based

Reply:
Copy-pasted this part brainless from EIP-2565 (last modification od ModExp pricing)